packaging/linux/run_keybase

```diff
@@ -13,6 +13,8 @@ set -e -u -o pipefail
 runtime_dir="${XDG_RUNTIME_DIR:-$HOME/.config}/keybase"
 mkdir -p "$runtime_dir"
 startup_token="$runtime_dir/startup_mode"
+mountdir="${XDG_DATA_HOME:-$HOME/.local/share}/keybase/fs"
```

Comment:
Should we do a `mkdir -p` here, as we do above on line 14?

Reply:
We have to do the `mkdir -p` after the unmount/kill of any previous mount, otherwise it'll fail.  I'll add a comment and move it below the `kill_all()` call.